src/Lookup/Lookup.tsx

```diff
@@ -65,6 +65,8 @@ function adjustOptionsAndSelected(selectedOption, options: any[]) {
 
 export type LookupOption = { value: any; label: string };
 
+export type AllowSameOptionSettings = { uniqueIdFieldName: any, originalIdSelector: (value: any) => any };
```

Comment:
Is `originalIdSelector` supposed to be different from `idSelectore` prop value?

Reply:
Yes, in a case when we need to support same options selection, idSelector should return some generated unique value(clientId etc)
Original idSelector instead select primary key of option, which will be same for same option selection